### PR TITLE
Change the guidelines about establishing base direction and mention ALM

### LINKED ISSUE #8: Change the rule for establishing base direction
http://w3c.github.io/bp-i18n-specdev/#bidi_block_change

I would make this rule a little clearer.

The content author must be able to indicate parts of the text where the base direction changes. At the block level, this should be achieved using attributes or metadata, and should not rely on Unicode control characters.

to

The content author must be able to indicate parts of the text where the base direction changes. At the block level, this should be achieved using attributes or metadata, and should not require the content author to use Unicode control characters to control direction.

### LINKED ISSUE #18: Should also mention ALM beside LRM and RLM
In 5.2.5 ALM (U+061C ARABIC LETTER MARK) should be mentioned as well:
- In the caption: "RLM and LRM" ==> replace with "RLM, ALM, and LRM"
- "A word about the Unicode characters U+200F RIGHT-TO-LEFT MARK (RLM) and U+200E LEFT-TO-RIGHT MARK (LRM) is warranted at this point."
  ==> "A word about the Unicode characters U+200F RIGHT-TO-LEFT MARK (RLM), U+061C ARABIC LETTER MARK (ALM), and U+200E LEFT-TO-RIGHT MARK (LRM) is warranted at this point (unless is used to force base direction when is estimated according to first-strong)."
- "The first point to be clear about is that neither RLM, nor LRM establish the base direction for a range of text.  They are simply invisible characters with strong directional properties."
  ==> "The first point to be clear about is that none of the RLM, ALM, or LRM establish the base direction for a range of text.  They are simply invisible characters with strong directional properties."
- "Of course, if you are detecting base direction using first-strong heuristics then RLM and LRM can be useful for setting the base direction where the text in question begins with something that would otherwise give the wrong result, eg. "
  ==> "Of course, if you are detecting base direction using first-strong heuristics then RLM, ALM, and LRM can be useful for setting the base direction where the text in question begins with something that would otherwise give the wrong result, eg. "

That said, it should be mentioned that "ALM is still not widely supported."

### PR DESCRIPTION
**Pull request by aphillips**
Fixes #8. Fixes #18.

This addresses both @stevenatkin's comment and @lkimmel's comment.

Note that I added "isolated" to the guideline about providing attributes in markup languages. I am tempted to want to add text to make isolating preferred to embedding. This section may need further review.

I also change the title of the section about strong marks, adding some text ("Strong directional marks:").

***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/bp-i18n-specdev/pull/136.html" title="Last updated on Oct 17, 2024, 3:32 PM UTC (e16baf4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/136/ba0a752...aphillips:e16baf4.html" title="Last updated on Oct 17, 2024, 3:32 PM UTC (e16baf4)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -875,13 +875,13 @@ <h4>Problems with control characters</h4>
...
-<p>The first point to be clear about is that neither RLM nor LRM establish the base direction for a range of text.&nbsp; They are simply invisible characters with strong directional properties.</p>
+<h4>Strong direction marks: RLM, LRM, and ALM</h4>
+<p>A word about the Unicode characters <span class="codepoint" translate="no"><img alt="RLM" src="images/200F.png"><code class="uname">U+200F RIGHT-TO-LEFT MARK</code></span> (RLM), <span class="codepoint" translate="no"><img alt="LRM" src="images/200E.png"><code class="uname">U+200E LEFT-TO-RIGHT MARK</code></span> (LRM), and <span class="codepoint" translate="no"><img alt="ALM" src="images/061C.png"><code class="uname">U+061C ARABIC LETTER MARK</code></span> (ALM) is warranted at this point.</p>
+<p>The first point to be clear about is that these three characters do not establish the base direction for a range of text.&nbsp; They are simply invisible characters with strong directional properties.</p>
```
Comment (xfq) [inline index.html]:
```suggestion
<p>The first point to be clear about is that these three characters do not establish the base direction for a range of text. They are simply invisible characters with strong directional properties.</p>
```

**Comment by xfq** [comment]
Overall LGTM, but we should probably link to some background information of these characters. We don't seem to have them our website (see https://github.com/w3c/i18n-drafts/issues/602). Maybe somewhere at unicode.org for now?

File: index.html [hunk 2]
```diff
@@ -875,13 +875,13 @@ <h4>Problems with control characters</h4>
 
 
 <section id="rlmlrm">
-<h4>RLM and LRM</h4>
-<p>A word about the Unicode characters <span class="codepoint" translate="no"><img alt="RLM" src="images/200F.png"><code class="uname">U+200F RIGHT-TO-LEFT MARK</code></span> (RLM) and <span class="codepoint" translate="no"><img alt="LRM" src="images/200E.png"><code class="uname">U+200E LEFT-TO-RIGHT MARK</code></span> (LRM) is warranted at this point.</p>
-<p>The first point to be clear about is that neither RLM nor LRM establish the base direction for a range of text.&nbsp; They are simply invisible characters with strong directional properties.</p>
+<h4>Strong direction marks: RLM, LRM, and ALM</h4>
```
Comment (aphillips) [inline index.html]:
```suggestion
<h4>Strong directional formatting characters: RLM, LRM, and ALM</h4>
```

**Review by r12a — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -875,15 +875,44 @@ <h4>Problems with control characters</h4>
...
-<p>The title is "<span dir="rtl">פעילות הבינאום, W3C</span>".</p>
+<h4>Strong directional formatting characters: RLM, LRM, and ALM</h4>
+<p>A word about the Unicode characters <span class="codepoint" translate="no"><img alt="RLM" src="images/200F.png"><code class="uname">U+200F RIGHT-TO-LEFT MARK</code></span> (RLM), <span class="codepoint" translate="no"><img alt="LRM" src="images/200E.png"><code class="uname">U+200E LEFT-TO-RIGHT MARK</code></span> (LRM), and <span class="codepoint" translate="no"><img alt="ALM" src="images/061C.png"><code class="uname">U+061C ARABIC LETTER MARK</code></span> (ALM) is warranted at this point.</p>
+<p>The first point to be clear about is that these three characters do not establish the base direction for a range of text. They are simply invisible characters with strong directional properties.</p>
+<p>This means that you cannot use RLM for example, to make the text <kbd>W3C</kbd> appear to the left of the Hebrew text in the following example.</p>
+<p>The title is "<span dir="rtl" lang="he">פעילות הבינאום, W3C</span>".</p>
```
Comment (xfq) [inline index.html:882]: When this text is rendered in the browser, "W3C" is already on the left. Should `dir="rtl"` be removed here?

**Pull request merged**